run-ib-relval.py

```diff
@@ -116,7 +116,7 @@ def process_relvals(threads=None, cmssw_version=None, arch=None, cmssw_base=None
             e = waitpid(p.pid, 0)[1]
             print("Time took to create jobs:", int(time() - stime), "sec")
             if e:
-                exit(e)
+                exit(0 if e == 0 else 1)
```

Comment:
@iarspider, sorry I am not able to understand this change.`waitpid(p.pid, 0)[1]` is already return `0` (in case of success) or `non-0`  (in case of failure) so whats wrong with `exit(e)`?

Reply:
Because `e` is not a valid exit code (e.g. it will be 256 if `runTheMatrix` exits after throwing exception), and `exit(256)` is treated as `exit(0)`:

```
$ python3 -c "exit(256)"; echo $?
0
```

Reply:
So `e` is either `-N` if signal N occurs, or `exit_code << 8` otherwise. We either decode it with `waitstatus_to_exitcode`, or just check for zero/non-zero.

Reply:
I get e>0 when kill signal is send to the process but yes if `exit(256)` causes python to exit with `0` then  current change is acceptable